docs/ref/django-admin.txt

```diff
@@ -364,6 +364,20 @@ standard output.
 When this option is set and ``--verbosity`` is greater than 0 (the default), a
 progress bar is shown in the terminal.
 
+Fixtures compression
+~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.2
+
+    Support for fixtures compression was added.
+
+The output file can be compressed in one of the ``gz``, ``bz2``, ``lzma``, or
```

Comment:
in --> into?

Reply:
"The file is compressed in a format" sounds right to me, "The file is compressed into a format" doesn't. But perhaps "compressed with a format" would be better.

Reply:
I've updated the sentence using "with".